.github/workflows/build.yml

```diff
@@ -93,6 +93,8 @@ jobs:
         with:
           configurationFile: ${{ matrix.configurationFile }}
           renovate-version: ${{ env.RENOVATE_VERSION }}
+          docker-cmd-file: example/entrypoint.sh
+          docker-user: root
```

Comment:
please add a seperate test job, so we can see both types

Reply:
done